test.sh

```diff
@@ -522,6 +522,19 @@ function test-build-and-run-client-server-perf-test()
     fi
 
     set +x
+
+    echo " "
+    echo "**** ${Me}: Client-server performance testing with L3-fprintf logging ON:"
+    echo " "
+    build-and-run-client-server-perf-test "${num_msgs_per_client}"  \
+                                          "${l3_log_enabled}"       \
+                                          "${l3_LOC_disabled}"      \
+                                          "fprintf"
```

Comment:
Previous refactoring that introduced `build-and-run-client-server-perf-test()` test-method now nicely serves us to add `"fprintf"` directive, to drive fprintf()-logging.